_posts/developers/apps/inrupt-tutorial/2020-11-06-00_first-app.md

```diff
@@ -80,127 +82,122 @@ npm install @inrupt/solid-client @inrupt/solid-client-authn-browser @inrupt/voca
+
+   ```shell
+   "dependencies": {
+      ...
+   },
+   "browserslist": [
+     "last 3 and_chr versions",
+     "last 3 chrome versions",
+     "last 3 opera versions",
+     "last 3 ios_saf versions",
+     "last 3 safari versions",
+     "last 3 edge versions"
+   ]
+   ```
+
+   Without the `browserslist` specification, Parcel would need to
+   inject a compatibility layer to avoid a `regeneratorRuntime` reference error
+   since our sample application uses `async` functions.
+
+   As an alternative to modifying the `package.json` file, you could import
+   `regenerator-runtime/runtime` in your JavaScript code, which will inject a
+   suitable compatibility layer.
 
 ### 4. Create the Application
 
 In the `my-demo-app` directory, create the files for the application.
 
 1. Create a `my-demo.css` file with the following content:
 
-    ```css
+   ```css
 
-    {% include_relative codesandbox/my-demo.css %}
+   {% include_relative codesandbox/my-demo.css %}
 
-    ```
+   ```
 
 1. Create an `index.html` file with the following content:
 
-    ```html
-
-    {% include_relative codesandbox/index.html %}
-
-    ```
+   ```html
+   {% include_relative codesandbox/index.html %}
+   ```
 
 1. Create an `index.js` file with the following content:
 
-    > **NOTE:** If your Pod is not on `https://solidcommunity.net`, modify the
-      value of the `SOLID_IDENTITY_PROVIDER` variable accordingly.
+   > **NOTE:** If your Pod is not on `https://solidcommunity.net`, modify the
+   > value of the `SOLID_IDENTITY_PROVIDER` variable accordingly.
 
-    ```javascript
+   ```javascript
 
-    {% include_relative codesandbox/index.js %}
+   {% include_relative codesandbox/index.js %}
 
-    ```
+   ```
 
 ## Run the Application
 
-1. In the ``my-demo-app`` directory, run:
+1. In the `my-demo-app` directory, run:
 
-    ```shell
-    npx parcel index.html
-    ```
+   ```shell
+   npx parcel index.html
+   ```
 
-    The output should resemble the following:
+   The output should resemble the following:
 
-    ```shell
-      Server running at http://localhost:1234
-      ...
-      Built in 2ms.
-    ```
+   ```shell
+     Server running at http://localhost:1234
+     ...
+     Built in 2ms.
+   ```
 
 1. Open `localhost:1234` in a browser.
 
 1. Click the `Login` button.
 
-    * The first time you log into your Pod with this application (it'll
-      be named `http://localhost:1234`), you'll
-      be prompted to authorize it to access your Pod . To allow the application to
-      read and write to your Pod, click `Authorize`.
-
-    * If you have logged out of your Pod, you are prompted to log in.
-      Enter your username and password to log in.
-
-    * Once logged in, you should be redirected back to the client application.
+   - The first time you log into your Pod with this application (it'll
+     be named `http://localhost:1234`), you'll
+     be prompted to authorize it to access your Pod . To allow the application to
```

Comment:
```suggestion
     be prompted to authorize it to access your Pod. To allow the application to
```

Reply:
Ditto.  Great catch.  But, same as before, since the content doesn't show ... not sure what the group prefers.